### ISSUE TITLE
About w18_v1 on cityscapes

### ISSUE THREAD

**Issue by zhao181** [HRNet/HRNet-Semantic-Segmentation#165]
Can any one offer an variation curve beteen miou and epoches on both train and validation set?
The website said that a 70.3 miou can be obtained. However, my best result is only 65.8% with an adam optimizier and exponential learning rate decay . 

With the official configuration, the miou on trainset will fluctuate around 0.56 and the validation miou is only around 0.52 after 100 epoches. So what should I do? Can any  offer an variation curve beteen miou and epoches? or give some suggestions? thanks a lot.
